coffee/chosen.jquery.coffee

```diff
@@ -510,3 +510,7 @@ class Chosen extends AbstractChosen
         w = f_width - 10
 
       @search_field.css({'width': w + 'px'})
+
+  form_field_has_changed: (extra) ->
```

Comment:
`trigger_form_field_change` was my first idea :smile:, I thought it might be best to be a bit less specific in the name, but I'm fine with that name too.
You don't use `_myPrivateMethod` for private/protected methods, right? It's not really a public method.